lambdas/services/create_document_reference_service.py

```diff
@@ -0,0 +1,131 @@
+import os
+
+from botocore.exceptions import ClientError
+from pydantic import ValidationError
+
+from enums.supported_document_types import SupportedDocumentTypes
+from models.nhs_document_reference import UploadRequestDocument, NHSDocumentReference
+from utils.audit_logging_setup import LoggingService
+from utils.exceptions import CreateDocumentRefException, InvalidResourceIdException
+from utils.lloyd_george_validator import validate_lg_files, LGInvalidFilesException
+from utils.utilities import create_reference_id, validate_id
+
+logger = LoggingService(__name__)
+
+
+class CreateDocumentReferenceService:
+    def __init__(self, nhs_number, s3_service_class, dynamo_service_class):
```

Comment:
Did we agree to set the base aws services as class variables for our custom services? Seems you're not really using them in the handler but creating them and then passing to this service